habitat-lab/habitat/articulated_agents/humanoids/kinematic_humanoid.py

```diff
@@ -73,6 +75,26 @@ def __init__(
         # a path, whereas the offset transform will be changing the position
         # to simulate different gaits
         self.offset_transform = mn.Matrix4()
+        self.offset_rot = -np.pi / 2
+        add_rot = mn.Matrix4.rotation(
+            mn.Rad(self.offset_rot), mn.Vector3(0, 1.0, 0)
+        )
+        perm = mn.Matrix4.rotation(
+            mn.Rad(self.offset_rot), mn.Vector3(0, 0, 1.0)
+        )
+        self.offset_transform_base = perm @ add_rot
+
+        self.rest_joints = None
+
+    def set_rest_pose_path(self, rest_pose_path):
```

Comment:
Why is this a publicly exposed method and the external agent manager is responsible for calling it? Can the KinematicHumanoid object just directly set the rest pose in its own constructor?

Reply:
Currently agents can only be passed the urdf_path. Is there a way we can pass more parameters? If so, I would pass the path to the resting pose to the constructor